README.md

```diff
@@ -9,7 +9,9 @@ following aspects:
 * inhibiting alerts based on alert dependencies
 * aggregating alerts by labelset
 * handling notification repeats
-* sending alert notifications via external services (currently email or [PagerDuty](http://www.pagerduty.com/))
+* sending alert notifications via external services (currently email,
+[PagerDuty](http://www.pagerduty.com/) or
+[Pushover](https://www.pushover.net/))
```

Comment:
I'm not sure if I understand correctly, the last of the parentheses are ending the one between `..services (currently email..` on row 12. Is that the one you mean?

Here's an image on how the markup renders:

![screen shot 2015-02-17 at 12 59 04](https://cloud.githubusercontent.com/assets/10686/6227852/067f6a1e-b6a5-11e4-80cf-205fdf0483c9.png)

Thanks!